content/en/docs/Concepts/olm-architecture/operator-catalog/creating-an-update-graph.md

```diff
@@ -255,6 +255,308 @@ entries:
       - myoperator.v0.3.0
 ```
 
+## Skip vs. SkipRange vs. Replaces by Example
+
+For these examples we will be considering an operator that is already installed (`Installed`) and how update options are selected for it.
+
+```mermaid
+graph LR
+  subgraph Update<br>Options
+    a
+    b
+    c
+  end
+  I(Installed) 
+  a -->|can update from| I
+  b -->|can update from| I
+  c -->|can update from| I
+```
+
+Invariants to keep in mind:
+
+- Anything in the channel that `replaces` the `Installed` package is an option.
```

Comment:
I'm not sure what you mean by `option`. I'm guessing that's because I do not have the graph in mind that you intended the reader to have a reference to.